lib/modem_info/modem_info.c

```diff
@@ -491,6 +493,13 @@ int modem_info_string_get(enum modem_info info, char *buf)
 
 	if (info == MODEM_INFO_ICCID) {
 		flip_iccid_string(buf);
+
+		/* Remove padding char from 19 digit (18+1) ICCIDs */
+		if ((len == ICCID_LEN) &&
+			(buf[len-1] == ICCID_PAD_CHAR)) {
```

Comment:
Please align with the opening bracket, right now it blends with the code in the block.